freedom-of-travel-impl/src/main/java/nc/project/service/AuthServiceImpl.java

```diff
@@ -0,0 +1,122 @@
+package nc.project.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import nc.project.auth.request.LoginRequest;
+import nc.project.auth.request.SignupRequest;
```

Comment:
Please remove the code which doesn't relate to the Login functionality

Reply:
done